### PR TITLE
Abr scale

### PR DESCRIPTION
**Pull request by rclarke0**
# Overview

Adding script to take and analyze scale measurements.

# Test Plan

I did not take any testing steps because this script is not used by any of the robots.

# Changelog

# Review requests

Double check that changes to the drivers on the scale do not break anything else.

# Risk assessment
low risk

### CODE REVIEW

**Review by andySigler — CHANGES_REQUESTED** [review]
thank you for this! let's sit down together and go through what needs change before a merge 👍

**Review by rclarke0 — COMMENTED** [review]
LGTM

File: hardware-testing/hardware_testing/scripts/abr_asair_sensor.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+"""ABR Temperature Humidity Sensors."""
+# from hardware_testing import data
```
Comment (andySigler) [inline hardware-testing/hardware_testing/scripts/abr_asair_sensor.py:2]: You can remove commented things. If it's commented out, then it should just be deleted.

Comment (andySigler) [reply hardware-testing/hardware_testing/scripts/abr_asair_sensor.py:2]: also should this file be a part of the PR? It's empty

**Review by andySigler — APPROVED** [review]
Looks ready to merge, other than that one file that seems empty other than the commented out pieces

**Pull request merged**